### PR TITLE
fix base64 invalid argument when running gce util.sh

### LINKED ISSUE #124240: Error: base64 invalid argument when running `kubetest gce --gcp-project myproject --up`
under kubernetes project and running 
```
kubetest gce --gcp-project myproject --up
```
get error:
```
base64: invalid argument /var/folders/zz/zyxvpxvq6csfxvn_n0000000000000/T/tmp.c9rh6GNDxr/easy-rsa-master/aggregator/pki/private/ca.key
Usage:	base64 [-Ddh] [-b num] [-i in_file] [-o out_file]
  -b, --break    break encoded string into num character lines
  -Dd, --decode   decodes input
  -h, --help     display this message
  -i, --input    input file (default: "-" for stdin)
  -o, --output   output file (default: "-" for stdout)
```
image

I am running on `macos m1` darwin arm64
running `base64 --help` get this:
```
base64 --help
Usage:	base64 [-Ddh] [-b num] [-i in_file] [-o out_file]
  -b, --break    break encoded string into num character lines
  -Dd, --decode   decodes input
  -h, --help     display this message
  -i, --input    input file (default: "-" for stdin)
  -o, --output   output file (default: "-" for stdout)
```

if running on linux:
```
Usage: base64 [OPTION]... [FILE]
Base64 encode or decode FILE, or standard input, to standard output.

With no FILE, or when FILE is -, read standard input.

Mandatory arguments to long options are mandatory for short options too.
  -d, --decode          decode data
  -i, --ignore-garbage  when decoding, ignore non-alphabet characters
  -w, --wrap=COLS       wrap encoded lines after COLS character (default 76).
                          Use 0 to disable line wrapping

      --help     display this help and exit
      --version  output version information and exit
```

### PR DESCRIPTION
**Pull request by chengjoey**
#### What this PR does / why we need it:
fix base64 invalid argument when running gce util.sh

#### Which issue(s) this PR fixes:
Fixes #124240 

#### Does this PR introduce a user-facing change?

```release-note
None
```

### CODE REVIEW

File: cluster/gce/util.sh [hunk 1]
```diff
@@ -20,6 +20,14 @@
 # config-default.sh.
 readonly GCE_MAX_LOCAL_SSD=8
 
+BASE64_HELP=$(base64 --help)
+BASE64_INPUT_FLAG=""
+if [[ $BASE64_HELP =~ "--input" ]]; then
+  BASE64_INPUT_FLAG="-i"
```
Comment (BenTheElder) [inline cluster/gce/util.sh:26]: let's use `--input` since we found it, instead of assuming it has the `-i` alias.

Comment (chengjoey) [reply cluster/gce/util.sh]: done

**Pull request merged**